lib/datadog/appsec/configuration/settings.rb

```diff
@@ -95,6 +95,46 @@ def self.add_settings!(base)
                 o.default DEFAULT_OBFUSCATOR_VALUE_REGEX
               end
 
+              settings :block do
+                settings :templates do
+                  option :html do |o|
+                    o.env 'DD_APPSEC_HTTP_BLOCKED_TEMPLATE_HTML'
+                    o.type :string, nilable: true
```

Comment:
We need to provide `nilable` because we use  `using_default?` when checking whether we need to use the default value. 

https://github.com/DataDog/dd-trace-rb/pull/2975/files#diff-83b38af05def4a7362ff8abda84f27d84964c9fce7781eb7c4a27ecbfc235c66R77

Since `using_default?` [computes the option](https://github.com/DataDog/dd-trace-rb/blob/8cf3b72d9f5079076eaa67533e62a6906bdd62e6/lib/datadog/core/configuration/options.rb#L93), we needed to provide something that would make it work. Another solution would be to give an empty string as default.